### DISCUSSION TITLE
Does the plugin work with postcss-nesting?

### DISCUSSION THREAD

**Discussion by ryanbelisle** [postcss/postcss-mixins#153] [Announcements]
Curious if the plugin can work with postcss-nesting instead of postcss-nested?

**Comment by ai** [comment]
It should

**Comment by adam-lynch** [comment]
I made https://github.com/postcss/postcss-mixins/pull/170 which adds a test for this

**Comment by adam-lynch** [comment]
It would be good if the README mentioned it supports this, like it does for postcss-nested. It doesn't need to give a code example, just a little mention

**Comment by ai** [comment]
Every PostCSS plugin works with other. Itv will be strange to keep allow-list.
